From db1faedd0784f7fc40add45bbfffd4007c07ab72 Mon Sep 17 00:00:00 2001
Message-Id: <db1faedd0784f7fc40add45bbfffd4007c07ab72.1423041670.git.chang-joon.lee@intel.com>
From: Uma Shankar <uma.shankar@intel.com>
Date: Tue, 13 Jan 2015 14:01:28 +0530
Subject: [PATCH 1/2] SQUASHME! [VPG]: drm/i915: Fixed Hotplug notification
 issue

Fixed HDMI hotplug event notification to audio driver during
connected boot. The WQ should be scheduled during HDMI audio
callback registration stage. This is not needed at every hotplug
detect, the notification logic in driver takes care of it.

SQUASHME!: This patch should be squashed with below patch:
	Author: Vandita Kulkarni <vandita.kulkarni@intel.com>
	Change-Id: I37d81ed68d310b2180bf06106bcae0766ca77b20
	FOR_UPSTREAM [VPG]: drm/i915: Add support for chv HDMI LPE audio

Issue: IRDA-2856
Change-Id: I4b106c642ae8d49e7a5700156272fea0cb64edb1
Signed-off-by: Uma Shankar <uma.shankar@intel.com>
---
 drivers/gpu/drm/i915/intel_hdmi.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_hdmi.c b/drivers/gpu/drm/i915/intel_hdmi.c
index 3fe5a79..e3f1dd8 100644
--- a/drivers/gpu/drm/i915/intel_hdmi.c
+++ b/drivers/gpu/drm/i915/intel_hdmi.c
@@ -1286,8 +1286,6 @@ intel_hdmi_detect(struct drm_connector *connector, bool force)
 #endif
 	}
 
-	schedule_work(&dev_priv->hdmi_audio_wq);
-
 #ifdef CONFIG_EXTCON
 	if (strlen(intel_connector->hotplug_switch.name) != 0) {
 		if (status == connector_status_connected)
-- 
1.7.9.5

